site.js

```diff
@@ -1314,7 +1422,7 @@ function generateCurse() {
 		randomFrom([
 			{
 				subjectText: randomFrom(["whale", "dolphin"]), 
-				chosen: function(){extemitiesName = "fins";},
+				chosen: function(){extemitiesName = "fins"; facialfeatureName = "dorsal fin";},

```

Comment:
The dorsal fin is more of a back feature than a facial one. May not matter, but you may want to replace with "blowhole" or something if you specifically mention face/head anywhere this is used.